From 08dacb632cc331027f39dcfa0b782aeb6f2f893a Mon Sep 17 00:00:00 2001
From: Stephen Gallagher <sgallagh@redhat.com>
Date: Tue, 2 Dec 2025 12:19:04 -0500
Subject: [PATCH 3/3] Restore error message

This was dropped in 4.0, but should be retained in RHEL 9 and 10 for
compatibility, particularly with existing tests that look for specific
messages.

Signed-off-by: Stephen Gallagher <sgallagh@redhat.com>
---
 src/sscg.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/sscg.c b/src/sscg.c
index 070d567bb189d42a20fd0a80f8fe2f7caae4d9eb..9f46cd622a4d55bd634a370ccc81ff063422b5af 100644
--- a/src/sscg.c
+++ b/src/sscg.c
@@ -361,6 +361,7 @@ main (int argc, const char **argv)
 done:
   if (ret != EOK)
     {
+      SSCG_ERROR ("%s\n", strerror (ret));
       if (options)
         {
           sscg_io_utils_delete_output_files (options->streams);
-- 
2.52.0

